### ISSUE TITLE
Stage CS root cert is now different than AMO root cert

### ISSUE THREAD

**Issue by hwine** [mozilla-services/autograph#598]
see [bug 1691797](https://bugzilla.mozilla.org/show_bug.cgi?id=1691797) for changes on 2021-02-12

`tools/autograph-monitor/xpi.go` only checks AMO

**Comment by g-k** [comment]
Yeah, I started working on this on Friday. I need to add more vars to the config object for the different AMO and CSig roots.
